packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -18,7 +18,7 @@ export const defaultServices = {
   },
 
   async handleSignUp(formData): Promise<any> {
```

Comment:
```suggestion
  async handleSignUp(formData) {
```

so that return type is auto-inferred from `Auth.signUp`

Reply:
👍